src/main/kotlin/app/revanced/patches/all/connectivity/wifi/spoof/SpoofWifiPatch.kt

```diff
@@ -1,27 +1,44 @@
-package app.revanced.patches.all.connectivity.wifi.spoof.patch
+package app.revanced.patches.all.connectivity.wifi.spoof
 
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
-import app.revanced.patcher.patch.annotations.Patch
-import app.revanced.patcher.patch.annotations.RequiresIntegrations
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
-import app.revanced.util.patch.*
+import app.revanced.util.patch.AbstractTransformInstructionsPatch
+import app.revanced.util.patch.IMethodCall
+import app.revanced.util.patch.Instruction35cInfo
+import app.revanced.util.patch.filterMapInstruction35c
 import com.android.tools.smali.dexlib2.iface.ClassDef
 import com.android.tools.smali.dexlib2.iface.Method
 import com.android.tools.smali.dexlib2.iface.instruction.Instruction
-import java.util.*
 
-@Patch(false)
-@Name("Spoof wifi connection")
-@Description("Spoofs an existing Wi-Fi connection.")
-@RequiresIntegrations
-class SpoofWifiPatch : AbstractTransformInstructionsPatch<Instruction35cInfo>() {
+@Patch(
+    name = "Spoof wifi connection",
```

Comment:
❓ : Capitalisation of Wi-Fi
> "Wi-Fi is a proper noun and a registered trademark. Capitalize and hyphenate when referring specifically to Wi-Fi technologies." https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi

```suggestion
    name = "Spoof Wi-Fi connection"
```